VideoLocker/src/main/java/org/edx/mobile/http/ThrottlingInterceptor.java

```diff
@@ -2,23 +2,25 @@
 
 import org.edx.mobile.logger.Logger;
 
+import java.io.IOException;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
-import retrofit.RequestInterceptor;
+import okhttp3.Interceptor;
+import okhttp3.Response;
 
 /**
  * TODO used for network control
  */
-public class ThrottlingInterceptor implements RequestInterceptor {
+public class ThrottlingInterceptor implements Interceptor {
     protected final Logger logger = new Logger(getClass().getName());
 
     private long lastRequest = 0L;
     private final long maxRequestSpeed = 1000;
     private final Lock requestLock = new ReentrantLock();
 
     @Override
-    public void intercept(RequestFacade request) {
+    public Response intercept(Chain chain) throws IOException {
```

Comment:
The interceptors are only used by the OkHttp client, and the library didn't add any nullability annotations, so the only point of adding them here would be to clarify expectations. Nor is this being done in the other interceptors. However, I'll add them in the new interceptors anyway.